src/quill-editor.component.ts

```diff
@@ -126,11 +127,11 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
 
     // update model if text changes
     this.quillEditor.on('text-change', (delta: any, oldDelta: any, source: string) => {
-      let html = this.editorElem.children[0].innerHTML;
+      let html: Html = this.editorElem.children[0].innerHTML;
```

Comment:
why you want to type this at all? If so then type it directly here as `html: string? = .....` optional string.

HTML is a false leading name --> HTMLString or something like this.